src/app/components/LabeledInput/LabeledInput.tsx

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import styles from "./LabeledInput.module.css";
+
+type LabeledInputProps = {
+  label: string;
+  value: string;
+  type?: string;
+  required?: boolean;
+  onChange: (value: string) => void;
+};
+
+function LabeledInput({
+  label,
+  value,
+  type,
+  required,
+  onChange,
+}: LabeledInputProps): JSX.Element {
+  return (
+    <div className={styles.container}>
+      <label className={styles.container__label}>{label}</label>
```

Comment:
Not a big issue, but you could connect the label with the input (if the input is a child of the label).
This makes it possible to click the label to select the input field.

Reply:
Good point, thanks! I connected them - maybe you could recheck?